From 265c608b8ae952ce52a1a40bd4161c6a69e4a365 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Sun, 06 Mar 2022 17:36:43 +0200
Subject: [PATCH] [SW-69581] gaudi3: fix spsram base address

Due to changes in ARC1 ICCM/DCCM size, spsram was moved.
updating correct address

Change-Id: I6bf8c70b4efaed3f68400be86c68762d4b7efd0c
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.h b/arch/gaudi3/arc/common.h
index a32e00d..57d6018 100644
--- a/arch/gaudi3/arc/common.h
+++ b/arch/gaudi3/arc/common.h
@@ -34,7 +34,7 @@
 #define SDRAM_BASE			(FW_MEM_REGION_BASE - SDRAM_SIZE)
 #define PCIE_FW_SRAM_BASE		(FW_MEM_REGION_BASE + 0x7F00000)
 #define FW_SRAM_BASE			(FW_MEM_REGION_BASE + 0x30E0000)
-#define SP_SRAM_BASE			(FW_MEM_REGION_BASE + 0x30D0000)
+#define SP_SRAM_BASE			(FW_MEM_REGION_BASE + 0x3120000)
 #define ARC1_ICCM_BASE			(FW_MEM_REGION_BASE + 0x3040000)
 #define ARC0_ICCM_BASE			(FW_MEM_REGION_BASE + 0x3000000)
 #define HL_SPI_BASE			(FW_MEM_REGION_BASE + 0x2000000)
